src/info/info_command.c

```diff
@@ -188,6 +188,23 @@ void fillReplyWithIndexInfo(RedisSearchCtx* sctx, RedisModule_Reply *reply, bool
           REPLY_KVSTR("distance_metric", VecSimMetric_ToString(hnsw_params.metric));
           REPLY_KVINT("M", hnsw_params.M);
           REPLY_KVINT("ef_construction", hnsw_params.efConstruction);
+        } else if (primary_params->algo == VecSimAlgo_SVS) {
+          REPLY_KVSTR("algorithm", VecSimAlgorithm_ToString(primary_params->algo));
+          SVSParams svs_params = primary_params->algoParams.svsParams;
+          REPLY_KVSTR("data_type", VecSimType_ToString(svs_params.type));
+          REPLY_KVINT("dim", svs_params.dim);
+          REPLY_KVSTR("distance_metric", VecSimMetric_ToString(svs_params.metric));
+          REPLY_KVINT("num_threads", svs_params.num_threads);
+          REPLY_KVINT("graph_max_degree", svs_params.graph_max_degree);
+          REPLY_KVINT("construction_window_size", svs_params.construction_window_size);
+          REPLY_KVINT("search_window_size", svs_params.search_window_size);
```

Comment:
"search_window_size" and "prune_to" should not be exposed at this point. see https://github.com/RedisAI/VectorSimilarity/pull/702#discussion_r2156988661